code/__HELPERS/spatial_info.dm

```diff
@@ -79,7 +79,8 @@
 /**
  * returns every hearaing movable in view to the turf of source not taking into account lighting
  * useful when you need to maintain always being able to hear something if a sound is emitted from it and you can see it (and youre in range).
- * otherwise this is just a more expensive version of get_hearers_in_LOS()
+ * otherwise this is just a more expensive version of get_hearers_in_LOS().
+ * uses hearers() instead of view() now as a slight optimization.
```

Comment:
crummy comment. what is the optimization, why do I care about how it used to work?